lib/application.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Application
+ * @constructor
+ * @param {Object} client Catapult client
+ */
+
+var Application = function (client) {
+	/**
+	 * List the user's applications
+	 * @param {Object} params Parameters for filtering applications
+	 * @param {Number} [params.page] The specified page requested when querying
```

Comment:
@fuchsnj I did not know that. We will discuss and update here. Are we going to update the documentation across the site?

So now people will search by `size` and then follow the `next` link in the header?